src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/VisitSchedulerClient.kt

```diff
@@ -247,6 +248,20 @@ class VisitSchedulerClient(
       .block(apiTimeout)
   }
 
+  fun getNotificationCountForPrison(prisonCode: String): NotificationCountDto {
+    return webClient.get()
+      .uri("/visits/notification/$prisonCode/count")
+      .retrieve()
+      .bodyToMono<NotificationCountDto>().block(apiTimeout)!!
```

Comment:
the !! will fail if there's a timeout. I think we need to return NotificationCountDto? instead.

Reply:
YehA never know how to deal with this as it's not optional... thought time out will throw an exception